templates/function-base/js/default/README.md

```diff
@@ -8,6 +8,10 @@ Azure Functions are a great way to add server-side behaviors to any Teams applic
 - An M365 account. If you do not have M365 account, apply one from [M365 developer program](https://developer.microsoft.com/en-us/microsoft-365/dev-program)
 - [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) or [TeamsFx CLI](https://aka.ms/teamsfx-cli)
 
+## Single Sign On
+
```

Comment:
We can say something like: `After adding function to your project, SSO will also be enabled. You can follow this [document](https://aka.ms/teamsfx-add-sso) to update your tab or bot code to include SSO.`